### PR TITLE
[HL-323] Create card component

### PR DESCRIPTION
**Pull request by arturwujczak**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -49,3 +50,4 @@ Thumbs.db
 *.wmv
 
 /all/target/
+/core/bulma.core.iml
```
Comment (nowak-pawel-ds) [inline .gitignore]: Line 28 is not covering that one?

**Review by marcell-pal-ds — APPROVED** [review]

File: ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                    name="thumbnail/src"
+                    description="Secondary image source"
+                    label="Use this if you want a link to image from web"/>
+            <alt
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="wcm/dialogs/components/textfield"
+                    name="thumbnail/alt"
+                    label="Alt text"/>
+        </thumbnail>
+        <mainImage
```
Comment (nowak-pawel-ds) [inline ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml:58]: thumbnail and mainImage have same fields, maybe you could create one dialog in artifact and use it twice just assigning it to different fields?

File: .gitignore (hunk 1 above)
Comment (arturwujczak) [reply .gitignore]: Right, the file was created earlier and manual ignoring resulted in adding this line

File: ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml (hunk 2 above)
Comment (arturwujczak) [reply ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml:58]: In the future, I would like additional options in these dialogs, such as size, and then they will not be the same. I mean, the size list will be different for the image and the thumbnail.

Comment (nowak-pawel-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/card/_ws_dialog/.content.xml:58]: https://teamds.atlassian.net/wiki/spaces/WEBS/pages/253067265/Include You could customize it with include function. So cut common part and then add new properties when using it.

**Review by nowak-pawel-ds — APPROVED** [review]

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**